src/main/java/teammates/ui/controller/InstructorFeedbackResponseCommentAddAction.java

```diff
@@ -81,6 +81,7 @@ protected ActionResult execute() throws EntityDoesNotExistException {
                 .withReceiverSection(response.recipientSection)
                 .withCommentFromFeedbackParticipant(false)
                 .withCommentGiverType(FeedbackParticipantType.INSTRUCTORS)
+                .withVisibilityFollowingFeedbackQuestion(false)
```

Comment:
Also we need to set `isVisibilityFollowingFeedbackQuestion` to false in instructor side.